Packs/Jira/Scripts/ScriptJiraListStatus/ScriptJiraListStatus.py

```diff
@@ -0,0 +1,100 @@
+import demistomock as demisto  # noqa: F401
+from CommonServerPython import *  # noqa: F401
+
+
+def create_output(error_msg):
+    return {
+        "Type": entryTypes["error"],
+        "ContentsFormat": formats["text"],
+        "Contents": error_msg,
+        "HumanReadable": error_msg,
+        "ReadableContentsFormat": formats["text"],
+    }
+
+
+def execute_xsoar_command(args: Dict[str, Any]) -> List[Dict[str, Any]]:
+    """Execute the command !jira-list-transitions with respect to the arguments `args`, which will hold the issue id
+    and which brand to use, Jira V3, or V2.
+
+    Args:
+        args (Dict[str, Any]): The arguments of the command.
+    Returns:
+        List[Dict[str, Any]]: The results of the executed command.
+    """
+    demisto.info(f'got the following args {args}')
+    res = demisto.executeCommand(
+        "jira-list-transitions", args
+    )
+    if res and isinstance(res, list):
+        return res
+    else:
+        raise DemistoException((f'Error occurred while running script-JiraListStatus, expected a list as response but got:'
+                               f' {type(res)}. The response is: {res}'))
+
+
+def extract_statuses_from_transition_response(command_execution_response: List[Dict[str, Any]]) -> List[str]:
+    """Gets the list of statuses from the CommandResults object returned by the command !jira-list-transitions when
+    using the brand Jira V3.
```

Comment:
I think that in this case, it is relevant to JiraV2 as well (according to your code).